### PR TITLE
edit regex to accept all 'mark' characters

### PR DESCRIPTION
**Pull request by yashnalla**

### CODE REVIEW

**Comment by yashnalla** [comment]
not ready, still writing tests

**Comment by yashnalla** [comment]
wrote a test, but couldn't really think of any others for this situation that is not already covered by other tests

File: app/utils.py [hunk 1]
```diff
@@ -206,15 +206,23 @@ def _split_string_tag(self, string_tag):
...
+        # %~ matches a single character in the list [%,~]
+        # + Quantifier — Matches between one and unlimited times, as many times as possible, giving back as needed
+        # the list of special characters that are not percent encoded is -._~()*!\' from Appendix D of RFC3986
+
+        groups = re.match(r"([\w.\-()*%~_~!\']+)\/([\w.\-()*%~_~!\']+)=([\w.\-()*%~_~!\']+)", string_tag)
         if groups:  # NS/key=value
             namespace, key, value = groups.group(1), groups.group(2), groups.group(3)
         else:
-            groups = re.match(r"([\w.\-%~]+)\/([\w.\-%~]+)", string_tag)  # NS/key
+            groups = re.match(r"([\w.\-()*%~_~!\']+)\/([\w.\-()*%~_~!\']+)", string_tag)  # NS/key
```
Comment (jharting) [inline app/utils.py]: Is the tilde char listed twice on purpose?

**Comment by Glutexo** [comment]
@yashnalla Because of the merge commits, the pull requests is now hadly reviewable. Please squash the changes and rebase on _master_.

**Comment by Glutexo** [comment]
Closing this as we have #596.

**Pull request closed without merging**